README.md

```diff
@@ -206,6 +206,9 @@ var options = {
   //custom class for unique track styling
   "customClass" : 'vocals',
 
+  //custom background-color for the canvas-drawn waveform
+  "waveBackgroundColor": "#f3f3f3",
+
```

Comment:
The playlist equivalent of this is `waveOutlineColor` it would be nice match this naming so it's very clear that these properties are for the same thing, but an override.